dspace-api/src/main/java/org/dspace/content/dao/impl/CollectionDAOImpl.java

```diff
@@ -159,7 +159,8 @@ public List<Collection> findAuthorizedByGroup(Context context, EPerson ePerson,
 
     @Override
     public List<Collection> findCollectionsWithSubscribers(Context context) throws SQLException {
-        return list(createQuery(context, "SELECT DISTINCT col FROM Subscription s join  s.collection col"));
+        return list(createQuery(context, "SELECT c FROM Collection c JOIN Subscription s ON c.id = " +
```

Comment:
Without using DISTINCT or GROUP BY there isn't any duplicated rows?

Reply:
I didn't notice duplicates in my testing, but you're correct that it's easy to prevent them. I added the DISTINCT function.